api/src/main/java/io/strimzi/api/kafka/model/CertSecretSource.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represents a certificate inside a Secret
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = true,
+        builderPackage = "io.strimzi.api.kafka.model"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CertSecretSource implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    protected String name;
+    protected String certificate;
+
+    protected Map<String, Object> additionalProperties;
+
+    @Description("The name of the Secret containing the certificate.")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String getName() {
```

Comment:
Just to consider ... wouldn't a more descriptive name such as `sercretName` be better here?